src/components/Toaster/index.tsx

```diff
@@ -8,25 +8,26 @@ export const Toast: any = (
   type: ToastProps['type'],
   title: ToastProps['title'],
   message?: ToastProps['message'],
+  position?: ToastProps['position'],
```

Comment:
By using default props, we can make this "required".

Reply:
ok